api/python/provisioner/commands/bootstrap/node_grains.py

```diff
@@ -0,0 +1,135 @@
+#
```

Comment:
@divya-vijay please either name that file as `node.py` or create a separate one for `Node` class

Reply:
Thanks! I've renamed it to `node.py`.